java/elemental2/dom/integer_entities.txt

```diff
@@ -92,13 +92,16 @@ elemental2.dom.DOMTokenList.length
 elemental2.dom.DOMTokenList.item.index
 elemental2.dom.DOMImplementationList.length
 elemental2.dom.DOMImplementationList.item.index
+elemental2.dom.Element.ALLOW_KEYBOARD_INPUT
```

Comment:
It seems that Element.ALLOW_KEYBOARD_INPUT doesn't exist anymore? We should deprecate it or even remove it.
I'll check if it used internally and delete the api if not

Reply:
Indeed it does look like it was deprecated and removed a while ago. I probably should have picked up in that. I was just poking around something else in that file and figured I would clean up as I went. :)